packages/oas/src/converter/parts/postdata/BodyConverter.ts

```diff
@@ -0,0 +1,188 @@
+import { Sampler } from '../Sampler';
+import { SubConverter } from '../../SubConverter';
+import { OpenAPI, OpenAPIV3, PostData } from '@har-sdk/core';
+import { toXML, XmlElement } from 'jstoxml';
+import { stringify } from 'qs';
+
+export abstract class BodyConverter<T extends OpenAPI.Document>
+  implements SubConverter<PostData | null>
+{
+  private readonly JPG_IMAGE = '/9j/7g=='; // 0xff, 0xd8, 0xff, 0xee
```

Comment:
```suggestion
  private readonly JPG_IMAGE = '/9j/4A=='; // 0xff, 0xd8, 0xff, 0xe0
```
as "[JFIF APP0 marker](https://en.wikipedia.org/wiki/JPEG_File_Interchange_Format#File_format_structure) (FF E0) segment immediately follows the SOI marker (FF D8)"

Reply:
@pmstss this corresponds to APP1 for Exif.

Reply:
@derevnjuk APP1 segment marker is 0xFF 0xE1 (which results in JPG_IMAGE = `/9j/4Q==`)

Reply:
@pmstss you are right, it's my fault. yet, as I mentioned above it is still a valid jpeg. The marker ending can be between 0xe0 and 0xef.